src/scratch/3d_flakser_del2/3d_flakser_2_nn.md

```diff
@@ -0,0 +1,214 @@
+---
+title: '3D-flaksar, del 2'
```

Comment:
3D-Flaksas, Del 2

Reply:
Rettskriving er små bokstavar. Vil heller fikse dei andre tre filene til å vere som denne.

Reply:
Okei. For det så litt rart ut når jeg hadde på bokmål og nynorsk i filteret og de så annerledes ut i listevisningen